src/autocomplete/StreamAutocomplete.js

```diff
@@ -14,11 +14,13 @@ class StreamAutocomplete extends Component {
   render() {
     const { filter, subscriptions, onAutocomplete } = this.props;
     const streams = subscriptions
-      .filter(x => x.name.toLowerCase().startsWith(filter.toLowerCase()))
-      .slice(0, 5);
```

Comment:
I have concerns that this will introduce a perf issue.
Rendering potentially thousands of items per keystroke will be slow.